### PR TITLE
Feature/v45

### PR DESCRIPTION
**Pull request by marsifi**
Updating to Cosmos SDK `v0.45.1`

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -493,17 +530,21 @@ func NewSifApp(
...
+		ibchost.ModuleName,
+		disptypes.ModuleName,
+		transferModule.Name(),
 		clptypes.ModuleName,
+		ethbridgetypes.ModuleName,
+		tokenregistrytypes.ModuleName,
 		oracletypes.ModuleName,
-		ethbridge.ModuleName,
-		dispensation.ModuleName,
-		tokenregistry.ModuleName,
```
Comment (AustinoBombino) [inline app/app.go:505]: What's the reason for removing these modules from the InitGenesis order?

File: app/app.go [hunk 2]
```diff
@@ -472,17 +475,51 @@ func NewSifApp(
...
+		authtypes.ModuleName,
+		banktypes.ModuleName,
+		govtypes.ModuleName,
+		crisistypes.ModuleName,
+		genutiltypes.ModuleName,
+		feegrant.ModuleName,
+		paramstypes.ModuleName,
+		vestingtypes.ModuleName,
 		ibchost.ModuleName,
-		dispensation.ModuleName,
```
Comment (AustinoBombino) [inline app/app.go:476]: Why is this being removed?

File: x/ethbridge/keeper/blacklist.go [hunk 3]
```diff
@@ -59,9 +59,11 @@ func (k Keeper) SetBlacklist(ctx sdk.Context, msg *types.MsgSetBlacklist) error
 
 func (k Keeper) GetBlacklist(ctx sdk.Context) []string {
 	var addresses []string
+
 	store := ctx.KVStore(k.storeKey)
-	iter := store.Iterator(types.BlacklistPrefix, nil)
+	iter := sdk.KVStorePrefixIterator(store, types.BlacklistPrefix)
```
Comment (AustinoBombino) [inline x/ethbridge/keeper/blacklist.go:64]: Does store.Iterator() need to be replaced throughout? I believe it also appears in the oracle keepr

File: app/app.go (hunk 2 above)
Comment (marsifi) [reply app/app.go:476]: It's still there, under `disptypes.ModuleName`

File: app/app.go (hunk 1 above)
Comment (marsifi) [reply app/app.go:505]: They are still there: under `ethbridgetypes.ModuleName`, `disptypes.ModuleName`, and `tokenregistrytypes.ModuleName`.

`ibctransfertypes.ModuleName` is still there too, as `transferModule.Name()`

File: x/ethbridge/keeper/blacklist.go (hunk 3 above)
Comment (marsifi) [reply x/ethbridge/keeper/blacklist.go:64]: Good point, I updated it in the oracle keeper as well, and extended the relevant unit test.
It doesn't appear anywhere else.

**Comment by Jedi2002** [comment]
Closing in favour of newer pull request 
[https://github.com/Sifchain/sifnode/pull/2819](url)

**Pull request closed without merging**